java6-utils/resources/report.dtd

```diff
@@ -0,0 +1,88 @@
+<!--
+  - Copyright 2000-2020 JetBrains s.r.o.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -   http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -->
+
+<!-- This DTD describes the JaCoCo XML report format. It is identified by the
```

Comment:
is it a copy of Jacoco DTD? if so, I'd ask in #copyright if it's ok to copy it as it's EPL-2

Reply:
There is no answer in #copyright so I replaced verification with DTD by comparison with golden XML files.
DTD was used only in tests so it isn't required to keep in in our repo.